### PR TITLE
1469573: purge orphaned pools - 9.54

### PR DESCRIPTION
**Pull request by vritant**

### CODE REVIEW

**Comment by vritant** [comment]
[Investigate DB script that can be run by liquibase to clean up rogue pools.](https://trello.com/c/uKtpqwWt/487-investigate-db-script-that-can-be-run-by-liquibase-to-clean-up-rogue-pools)

File: server/src/main/resources/db/changelog/20170721050024-purge-orphaned-pools.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <changeSet id="20170721050024-1" author="vrjain">
+        <comment> purge orphaned pools</comment>
+        <sql>
+            delete from cp_pool where id in
+            (  select p.id
+               from cp_pool p
+                   left outer join cp_pool_source_stack ss
+               on p.id = ss.derivedpool_id
+               where p.type='STACK_DERIVED'
+                 and ss.derivedpool_id is null
```
Comment (Ceiu) [inline server/src/main/resources/db/changelog/20170721050024-purge-orphaned-pools.xml]: This won't match anything.

Comment (vritant) [inline server/src/main/resources/db/changelog/20170721050024-purge-orphaned-pools.xml]: @crog why so? lets chat online.

**Comment by vritant** [comment]
@mstead please let me know if we should close this PR against a KB article.

**Comment by mstead** [comment]
After speaking with mccune, we decided to go with a script + Kbase instead of an automatic cleanup due to complexity and added risk. Closing.

**Pull request closed without merging**